includes/polling/os/exablaze-fusion.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+#EXALINK-FUSION-MIB::fusionInfoSerial = STRING: EXAFSN-B-00402
+#EXALINK-FUSION-MIB::fusionInfoVersion = STRING: 1.5.0
+#EXALINK-FUSION-MIB::fusionInfoBoard = STRING: EBFSN-03
+#EXALINK-FUSION-MIB::fusionInfoSoftware = STRING: 2016-09-05 18:16:58 +1000 (c4573c4)
+
+$hardware = snmp_get($device, "fusionInfoBoard", "-Ovq", "EXALINK-FUSION-MIB");
```

Comment:
Please convert these four `snmp_get()` calls to a single `snmp_get_multi_oid()` call.